src/UglyToad.PdfPig/Export/ITextExporter.cs

```diff
@@ -0,0 +1,17 @@
+﻿using UglyToad.PdfPig.Content;
+
+namespace UglyToad.PdfPig.Export
+{
+    /// <summary>
+    /// Exports the page's text into the desired format.
+    /// </summary>
+    public interface ITextExporter
+    {
+        /// <summary>
+        /// Get the text representation.
+        /// </summary>
+        /// <param name="page"></param>
+        /// <returns></returns>
+        string Get(Page page);
+    }
```

Comment:
I'd suggest a method on here for exporting the whole document, if that's a supported feature of these formats, or is each XML/HTML document only valid on a per page basis?

```suggestion
        string ExportDocument(PdfDocument document);
    }
```

Reply:
This is a good question and the answer is not clear to me. It seems PageXml only support per page basis, and hOCR and Alto support both. The problem with the whole document approach with all methods is that they use a reference image of the page, so that bounding boxes can be drawn on the picture (see the hOCR html file rendering with _hocrjs_). 

So the whole document approach will not be rendered correctly.